arch/arm64/boot/dts/qcom/sm7150-samsung-a715f.dts

```diff
@@ -606,10 +606,10 @@
 
 		/*
 		 * In the downstream kernel pin 11 of &tlmm is used for resetting the display.
-		 * If it's actually set here then the display will turn off as soon as the pin is set to 0 and apparently it won't turn on again.
+		 * If it's actually set here then the display will turn off as soon as the pin is set to LOW and apparently it won't turn on again.
 		 * If it's not set at all the display won't show anything because reset-gpios is required by the driver.
 		*/
-		reset-gpios = <&tlmm 9 GPIO_ACTIVE_HIGH>; // TODO: Check this again or fix in driver
+		reset-gpios = <&tlmm 11 GPIO_ACTIVE_HIGH>;
```

Comment:
your reset gpio is almost certainly active low and that should fix all your problems